README.md

```diff
@@ -46,7 +46,7 @@ echo "error page" >> client/dist/error.html
 **Fourth**, run the plugin, and visit your new website!
 
 ```
-serverless client deploy [--stage $STAGE] [--region $REGION]
+serverless client deploy [--stage $STAGE] [--region $REGION] [--no-delete-contents] [--no-config-change] [--no-policy-change] [--no-cors-change]
```

Comment:
Does the `no` in all the flags get removed somewhere?

These options make sense to me. Just have to figure out how to refactor some of the documentation to explain these options. And also decide if these make sense as command line args or as config details in serverless.yml.